### ISSUE TITLE
shuffle=False in training data loader?

### ISSUE THREAD

**Issue by ChenyangLiu** [zzangjinsun/NLSPN_ECCV20#24]
Hi, nice work! I'm wondering that why didn't you set the training data loader to be shuffled?

https://github.com/zzangjinsun/NLSPN_ECCV20/blob/ba33fa5d9ea62ca970026a145ab18fab76d79d4a/src/main.py#L101

**Comment by zzangjinsun** [comment]
Hi @ChenyangLiu,

as far as I know, **DistributedSampler** does shuffle so DataLoader shuffle is set to False.

https://github.com/zzangjinsun/NLSPN_ECCV20/blob/ba33fa5d9ea62ca970026a145ab18fab76d79d4a/src/main.py#L93-L96

Please refer to [here](https://pytorch.org/docs/stable/data.html#torch.utils.data.distributed.DistributedSampler).

**Comment by ChenyangLiu** [comment]
Thanks for your reply!
